api/dp-openapi.yaml

```diff
@@ -917,6 +917,137 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /projects/{projectid}/checkout:
+    put:
+      tags:
+      - project
+      description: Check out a page for proofreading. Response - schema - proof_page.
```

Comment:
`Response - schema - proof_page.`?

Reply:
Removed this since no amiguity in this case.